packages/wonder-blocks-clickable/components/clickable.js

```diff
@@ -329,4 +346,10 @@ const styles = StyleSheet.create({
         WebkitFontSmoothing: "inherit",
         MozOsxFontSmoothing: "inherit",
     },
+    link: {
+        cursor: "pointer",
+    },
+    focused: {
```

Comment:
Clickable is used by other WB components, right? How does this change impact them?

Reply:
`ClickableBehavior` is used in other components, but not `Clickable`.